persistence/rocksdb/src/test/resources/11_0_x_rocksdb_data/data/LOG

```diff
@@ -0,0 +1,256 @@
+2020/11/23-10:53:43.160947 7f5a5c2ea700 RocksDB version: 6.8.1
```

Comment:
should not commit

Reply:
Same applies to everything from the data/ folder I suppose

Reply:
We need to commit the `data/` dir as this contains the `11.0.x` which we migrate to the new format on startup in the `RocksDBStoreCompatibilityTest`